Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -2002,31 +1997,17 @@
     CGContextStrokePath();
 
     // Lock the data that is backing the layer.
-    BYTE* buffer = static_cast<BYTE*>(_imgDest->Backing()->LockImageData());
-    THROW_NS_IF_NULL(E_UNEXPECTED, buffer);
-    int stride = _imgDest->Backing()->BytesPerRow();
-    float width = _imgDest->Backing()->Width();
+    _imgDest->Backing()->LockImageData();
```

Comment:
OK now who own the lock after the the GetRenderTarget() has returned? Someone needs to say the RenderTarget is being used so don't free it and that is what is done by this lock

---
In reply to: [80825385](https://github.com/Microsoft/WinObjC/pull/1040#discussion_r80825385) [](ancestors = 80825385,80797038,80771507)